### DISCUSSION TITLE
Is there a plan for distributed support?

### DISCUSSION THREAD

**Discussion by sgf** [microsoft/FASTER#933] [General]
The lack of distributed support has seriously affected the use scenarios of FASTER.

Of course, in the open source community, there is actually no shortage of distributed key-value storage related projects in the rust, c++, and java ecosystems.

But the C# ecosystem is very scarce, and it is still blank at present.
Therefore, it may be of great significance to implement this plan.

**Comment by TechnikEmpire** [comment]
https://github.com/microsoft/garnet
